### PR TITLE
fix: Add Nullish check to convert lists and tables

### PR DESCRIPTION
**Pull request by hata6502**
This pull request fixes two TypeErrors that occur under specific conditions when converting HTML elements to Cosense:

1. Unordered lists
> TypeError: Cannot read properties of undefined (reading 'map') at processList (/usr/src/app/node_modules/html2sb-compiler/dist/toScrapbox.js:80:10) at /usr/src/app/node_modules/html2sb-compiler/dist/toScrapbox.js:115:27 at Array.map (<anonymous>) at processList

2. Tables
> TypeError: Cannot read properties of undefined (reading 'map') at /usr/src/app/node_modules/html2sb-compiler/dist/toScrapbox.js:141:44 at Array.map (<anonymous>)

## Changes

Added nullish check in `packages/html2sb-compiler/src/toScrapbox.ts` to convert lists and tables.
`children` property is defined as nullable in `parse.ts`, so it is also assumed to be nullable in `toScrapbox.ts`.

## Checklists

- [x] Add some test cases for this change
- [x] Pass `pnpm run test` in `packages/html2sb-compiler` directory
- [x] Pass `pnpm run lint` in `packages/html2sb-compiler` directory

### CODE REVIEW

File: packages/html2sb-compiler/test/fixtures.test.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import fs from "node:fs";
 import path from "node:path";
-import { expect, test } from 'vitest'
+import { expect, test } from "vitest";
```
Comment (hata6502) [inline packages/html2sb-compiler/test/fixtures.test.ts:3]: This change is not essential, but was caused by `pnpm run lint:fix`.

File: packages/html2sb-compiler/test/fixtures/invalid-list-in-list.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<ul><ul></ul></ul>
```
Comment (hata6502) [inline packages/html2sb-compiler/test/fixtures/invalid-list-in-list.html:1]: `<ul>` can't be in `<ul>` as HTML syntax rule..., but this test case is added as [Forgiving Parsing](https://en.wikipedia.org/wiki/Tag_soup#:~:text=XHTML%20article.-,HTML5).

By this pull request, web pages such as https://faq.airpayment.jp/hc/ja/articles/360013687633 will be converted as much as possible.
[Image from Gyazo](https://nota.gyazo.com/a58f091da632ec690cf79b094eab2fad)

Comment (pastak) [reply packages/html2sb-compiler/test/fixtures/invalid-list-in-list.html:1]: good  as an edge case 😄

**Review by pastak — APPROVED** [review]
LGTM
